pgjdbc/src/main/java/org/postgresql/core/NativeQuery.java

```diff
@@ -27,13 +28,14 @@
   }
 
   public NativeQuery(String nativeSql) {
-    this(nativeSql, NO_BINDS);
+    this(nativeSql, NO_BINDS, false);
   }
 
-  public NativeQuery(String nativeSql, int[] bindPositions) {
+  public NativeQuery(String nativeSql, int[] bindPositions, boolean compatible) {
```

Comment:
@polobo , when you say "public API", you mean "public to pgjdbc, not public to everyone", don't you?